pkgs/tools/graphics/map-machine/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, fetchFromGitHub, python3, pkgs }:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "map-machine";
+  version = "dcf3ffa00902d7812dd581c959e97d8d33f3a78d";
+
+  src = fetchFromGitHub {
+    owner = "enzet";
+    repo = pname;
+    rev = version;
+    sha512 = "sha512-VtdsI6ATIWoksLzUQGEVlcIjQ9ADrBQwfmUPPHI/DHRcC8yKKiiiKuSxF5mOphKIyWVCL065iMdnzRCccI8QLg==";
+  };
+
+  doCheck = false;
```

Comment:
Is it possible to enable a subset/disable some tests that would succeed?

Reply:
I currently can't figure out a way to elegantly do that, as the failure occurred not in a test, but in the loading of the tests. This is due to code only being used in building the documentation using an obscure library called "moire".